src/SerilogWeb.Classic/Classic/Enrichers/HttpRequestClientHostIPEnricher.cs

```diff
@@ -69,10 +69,10 @@ public void Enrich(LogEvent logEvent, ILogEventPropertyFactory propertyFactory)
             if (HttpContext.Current == null)
                 return;
 
-            if (HttpContext.Current.Request == null)
+            if (HttpContextCurrent.Request == null)
```

Comment:
In this case it seems sufficient to remove the first check. I miss null coalescing operator from C# 6.0 so much!